core/src/test/java/org/apache/accumulo/core/conf/HadoopCredentialProviderTest.java

```diff
@@ -150,16 +154,9 @@ public void testConfigurationCreation() {
 
   @Test
   public void createKeystoreProvider() throws Exception {
-    File targetDir =
-        java.nio.file.Path.of(System.getProperty("user.dir")).resolve("target").toFile();
-    File keystoreFile = targetDir.toPath().resolve("create.jks").toFile();
-    if (keystoreFile.exists()) {
-      if (!keystoreFile.delete()) {
-        log.error("Unable to delete {}", keystoreFile);
-      }
-    }
```

Comment:
I'm a little confused why the file deletion was removed from these changes. It seems like the file is supposed to get re-created, or something, and this cleaned up the old one if it existed from a previous build. My guess is that behavior is still desired.